Kwenta/kwenta
# Fix the missing slider trail

## Description
Fix the following issues:
1. Leverage slider is not visible on the desktop.
2. Move the leverage value above the leverage slider on mobile.
3. Fix the displayed value of the slider label
4. When the user reaches the maximum leverage, the `up to xx` and max leverage label will show the negative value (in this case, the slider is disabled). The original solution is trying to use the `10x` as a placeholder but it's not working as expected because it only uses the placeholder value when the value is null (in this case, it's a negative value). I changed the logic a little bit to use the `10x` as the placeholder when the slider is disabled.

## Related issue
#1002 

## Motivation and Context
Improved the UI

## How Has This Been Tested?
1. Open the market page on the desktop and see the slider rail
2. Open the market page on the mobile and see the leverage value above the leverage slider

## Screenshots (if appropriate):
![image](https://user-images.githubusercontent.com/4819006/176973733-48428cc4-f959-4ee4-aafe-c2f22f14beda.png)
![image](https://user-images.githubusercontent.com/4819006/176973770-64b7608e-c274-4d04-a071-e787f619f10c.png)
![image](https://user-images.githubusercontent.com/4819006/176975715-d63e0df8-5734-407a-a81d-6bf1d55cba70.png)
